### PR TITLE
Update testing-suggestions.md

### PR DESCRIPTION
**Pull request by lukehansell**
I've added a couple of **bad**/**better** examples for enzyme testing that I've noticed while refactoring tests.

### CODE REVIEW

File: testing-suggestions.md [hunk 1]
```diff
@@ -403,6 +403,82 @@ As you can see our `beforeEach` block is now much simpler. With the help of [Enz
 
 This is just one example of using [Enzyme](https://airbnb.io/enzyme) to make React testing more readable and concise. I would encourage everyone using [Enzyme](https://airbnb.io/enzyme) to read the well written [documentation](http://airbnb.io/enzyme/docs/api/index.html).
 
+#### Best Practices
+**Bad**
+```
```
Comment (theninj4) [inline testing-suggestions.md]: Can we get ```javascript on these blocks for syntax highlighting

Comment (lukehansell) [reply testing-suggestions.md]: good call, have done 👍

**Comment by cameronviner** [comment]
Looks awesome to me +1

File: testing-suggestions.md [hunk 2]
```diff
@@ -403,6 +403,82 @@ As you can see our `beforeEach` block is now much simpler. With the help of [Enz
...
+  beforeEach(() => {
+    instance.handleClick();
+  });
+
+  it('does something', () => {
+    ...
+  });
+});
+```
+By testing the instance of the method we check that the method does what is expected, but by not checking it on the child component we are not testing it's integration within the child's context and that the component receives the function as intended.
```
Comment (GeorgeF72) [inline testing-suggestions.md:462]: 👍

**Review by GeorgeF72 — APPROVED** [review]
Good stuff

**Pull request merged**